docs/event.md

```diff
@@ -6,9 +6,9 @@
 
 This allows a low barrier to entry for anyone wanting to submit events to Spoor.
 
-__Important__: To decide how to name the category and action fields, consider this sentence (square brackets denote that part is optional):
+__Important__: To decide how to name the category and action fields, consider this sentence (angle brackets denote that part is optional):
 
-*A user can {action} a {category}[ with/having a {context}[ to {destination}]]*
+*A user can {action} a {category}< with/having a {context}< to {destination}>>*
```

Comment:
does it allow using `\[` to mean a literal squacket?

Reply:
\[like this\](https://i-am-not-a-link.com)

Reply:
That does work :-)

Reply:
I've done that in https://github.com/Financial-Times/o-tracking/pull/289/commits/7f3fce7633ce04710f9b2e499831711546b1ea31 now :-)